GoogleForCreators/web-stories-wp
# Select page after detaching background media

## Context



## Summary
Currently, when moving background media, the media stays selected. This PR changes the selection to Page for keeping the same panels.


## Relevant Technical Choices



## To-do
N/A


## User-facing changes



## Testing Instructions



1. Add background media
2. Detach media from the background using `X` in the Page background panel.
3. Verify that the Page is selected and the Page background panel is still visible.
### QA
Use the same steps as above.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #6907
